tensorboard/backend/application.py

```diff
@@ -199,6 +208,7 @@ def TensorBoardWSGIApp(
     deprecated_multiplexer: Optional `plugin_event_multiplexer.EventMultiplexer`
         to use for any plugins not yet enabled for the DataProvider API.
         Required if the data_provider argument is not passed.
+    extra_routes: See TBContext documentation for more information.
```

Comment:
Let's put the main docs here rather than TBContext, since it's only on TBContext as a hack to shoehorn it into CorePlugin.

Also, let's maybe name it `experimental_extra_routes` and/or add a warning in the docstring to indicate that it's not a stable API.